editor/src/components/canvas/canvas-strategies/strategies/absolute-flex-reparent-strategy-helpers.spec.browser2.tsx

```diff
@@ -2156,7 +2156,7 @@ describe('Reparent indicators', () => {
     await renderResult.getDispatchFollowUpActionsFinished()
 
     // Check the indicator presence and position.
-    await checkReparentIndicator(renderResult, 432, 637, 2, 123)
+    await checkReparentIndicator(renderResult, 388, 645, 2, 123)
```

Comment:
FYI to others viewing this - these hard coded values are based on exact pixel coordinates, and so have all had to be manually offset as a result of the menubar change. It seems we have quite a few tests along these lines, which we need to address separately using values calculated based on actual DOM elements.